src/test/java/com/github/dockerjava/netty/exec/ExecStartCmdExecTest.java

```diff
@@ -193,16 +197,22 @@ public void execStartNotAttachedStdin() throws Exception {
         ExecCreateCmdResponse execCreateCmdResponse = dockerClient.execCreateCmd(container.getId())
                 .withAttachStdout(true)
                 .withAttachStdin(false)
-                .withCmd("/bin/sh").exec();
+                .withCmd("/bin/sh")
+                .exec();
 
         boolean completed = dockerClient.execStartCmd(execCreateCmdResponse.getId())
                 .withDetach(false)
                 .withStdIn(stdin)
                 .exec(new ExecStartResultCallback(stdout, System.err))
                 .awaitCompletion(5, TimeUnit.SECONDS);
 
-        // with v1.22 of the remote api the server closed the connection when no stdin was attached while exec create, so completed was true
-        assertFalse(completed, "The process was not finished.");
         assertEquals(stdout.toString(), "");
+
+        if (getVersion(dockerClient).isGreaterOrEqual(VERSION_1_23)) {
```

Comment:
That one of the ways, but i decided to make conditionals in tests. Would also help understand for end users tests as examples.
With groups/tags you can only exclude Test from run, here it only checks behaviour.